src/index.html

```diff
@@ -7,8 +7,93 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="icon">
+          <a href="#Home">
+            <img
+             src="images\logo.png"
+             alt="logo"
+             class="logo">
```

Comment:
```suggestion
            <img
               src="images\logo.png"
               alt="logo"
               class="logo"
             >
```
https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping